spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -371,18 +365,13 @@ private SparkContext createSparkContext_2() {
   }
 
   public SparkContext createSparkContext_1() {
-    logger.info("------ Create new SparkContext {} -------", getProperty("master"));
-
-    String execUri = System.getenv("SPARK_EXECUTOR_URI");
-    String[] jars = null;
-
-    if (Utils.isScala2_10()) {
```

Comment:
Could you elaborate? seem like removing this could break some environment with jar already added to repl?